src/client/kinto_http.rs

```diff
@@ -76,16 +76,16 @@ impl std::fmt::Display for ErrorResponse {
     }
 }
 
-pub async fn get_latest_change_timestamp(
-    requester: &'_ (dyn Requester + 'static),
+pub fn get_latest_change_timestamp(
+    requester: &Box<dyn Requester + 'static>,
```

Comment:
Why wrap this in a Box?
https://rust-lang.github.io/rust-clippy/master/index.html#/borrowed_box

Reply:
You right, they were removed in https://github.com/mozilla-services/remote-settings-client/commit/c6305db63d308cad76b570539f3c8241aed624d7

i will backport these changes